obs-packaging/create-repo-branch.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+[ -z "${DEBUG}" ] || set -o xtrace
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+script_name="$(basename "${BASH_SOURCE[0]}")"
+
+# shellcheck source=./../scripts/lib.sh
+source "${script_dir}/../scripts/lib.sh"
+
+# shellcheck source=scripts/obs-docker.sh
+source "${script_dir}/scripts/obs-docker.sh"
+
+readonly home_project="home:katacontainers"
+readonly template_pkg="kata-pkg-template"
+arch_target=${ARCH:-$(uname -m)}
+
+# shellcheck source=scripts/obs-docker.sh
+source "${script_dir}/scripts/obs-pkgs.sh"
+
+pkg_exist() {
+	local project="$1"
+	local pkg="$2"
+
+	docker_run osc list "${project}" | grep "${pkg}" || return 1
+	return 0
+
+}
+
+# Array of repositories.
+#
+# Each element is comprised of mutiple parts in the form:
+#
+#   name::project::repository
+#
+typeset -a repos
+repos+=(CentOS_7::CentOS:CentOS-7::standard)
```

Comment:
I do wonder if it would be better to read the repos (and maintainers below) from a config file rather than having them inline here. That "config file" could be as simple as a `data.sh` that is `source`'d, or a full-blown YAML file.